src/javascripts/components/myNavbar/myNavbar.js

```diff
@@ -4,8 +4,34 @@ import 'firebase/auth';
 const logoutEvent = () => {
   $('#navbar-logout-button').on('click', (e) => {
     e.preventDefault();
+
+    // NOTE FOR STUDENTS 
+    // Remove session storage if they log out in the same session and in case another user logs in, we want the API check to happen.
+    window.sessionStorage.removeItem('ua');
```

Comment:
Removing the session storage item that was set on login so that when they login again, we can make a call to the API to check if they are already a user in our DB